examples/asr/transcribe_speech.py

```diff
@@ -387,10 +401,22 @@ def autocast(dtype=None):
 
     # transcribe audio
 
-    amp_dtype = torch.float16 if cfg.amp_dtype == "float16" else torch.bfloat16
+    if cfg.calculate_rtfx:
+        total_duration = 0.0
+
+        with open(cfg.dataset_manifest, "rt") as fh:
+            for line in fh:
+                item = json.loads(line)
+                if "duration" not in item:
+                    raise ValueError(
```

Comment:
Revert this, don;t raise error but only warning to say that it is recommended to have duration in manifest for faster inference!

Reply:
No. It's behind the calculate_rtfx flag, which defaults to false. If someone passes in a manifest without durations, transcribe_speech.py will still continue to run.